game/script-topics.rpy

```diff
@@ -5344,6 +5347,135 @@ label talk_fighting_drowsiness:
+        n 1fslem "Not like it'd be {i}just{/i} tea anyway,{w=0.1} knowing a jerk like {i}you{/i}."
+
+        return
+
+    n 1fllbo "..."
+    n 1fcseml "T-{w=0.1}that's not to mean I think it sucks,{w=0.1} or something like that!"
+    n 1nlrbo "I just have my own tastes."
+
+    if get_topic("talk_favourite_drink").shown_count > 0:
+        extend 1uspbg " Like hot chocolate!"
+
+    n 1ullaj "But...{w=0.5}{nw}"
+    extend 1nllbo " I guess I can see why people are into it so much."
+    n 1tnmca "Tea contains caffeine,{w=0.1} right?{w=0.5}{nw}"
+    extend 1tlrss " Not as much as coffee or anything,{w=0.1} but an edge is still an edge,{w=0.1} I guess."
+    n 1unmaj "It comes in a whole bunch of flavours too!{w=0.5}{nw}"
+    extend 1unmgs " I was actually kinda surprised at the variety!"
+    n 1ullss "You've got your regular old black tea{w=0.3}{nw}"
+    extend 1fslss " -{w=0.1} obviously -{w=0.3}{nw}" 
+    extend 1ulraj " but you've got green tea,{w=0.1} herbal tea..."
+    n 1uspgs "Even flavoured ones like cinnamon and peppermint!"
+    n 1nslss "We only ever had oolong tea in the clubroom though,{w=0.5}{nw}"
+    extend 1tnmss " so who knows?"
+    n 1ulrbo "Maybe I'd warm up to it if I tried some that sounded good."
+    n 1ulraj "But...{w=0.5}{nw}"
+    extend 1nslss " I've gone on enough." 
+    n 1unmbo "What about you,{w=0.1} [player]?"
+
+    if not persistent.jn_player_tea_coffee_preference:
+        menu:
+            n "What's your preference?"
+
+            "I prefer tea.":
+                $ persistent.jn_player_tea_coffee_preference = "tea"
+                n 1unmaj "Tea?{w=0.5}{nw}"
+                extend 1nllpu " Hmm..."
+                n 1unmbo "Yeah,{w=0.1} that's about what I expected."
+                n 1nlrbo "..."
+                n 1tnmbg "What?"
+                n 1tsqbg "Not like you raised a fuss about it earlier,{w=0.5}{nw}"
+                extend 1tsqsm " right?"
+                n 1kslsm "..."
+                n 1nslss "Though...{w=1.5}{nw}"
+                extend 1uslsr " not like you had much of a choice in it back then,{w=0.1} huh?"
+
+            "I prefer coffee.":
+                $ persistent.jn_player_tea_coffee_preference = "coffee"
+                n 1unmaj "Oh?{w=1.5}{nw}"
+                extend 1tnmss " You're a coffee drinker?"
+                n 1nllpu "Hmm..."
+                n 1nsqss "Then I guess the sessions in the club weren't your{w=0.5}{nw}"
+                extend 1fsqbg " {i}cup of tea{/i}{w=0.5}{nw},"
+                extend 1usqbg " huh?"
+                n 1uchgn "..."
+                n 1fchbg "Oh,{w=0.5}{nw}" 
+                extend 1fllbg " come on,{w=0.1} [player]!{w=0.2} Yeesh."
+                n 1fsqsm "No need to be all {w=0.3}{i}bitter{/i}{w=0.3} about it."
+                n 1fchsm "..."
+                n 1kchbg "Okay,{w=0.1} okay!{w=0.5} I'm done!"
+                n 1fsqsg "...For now."
+
```

Comment:
```suggestion

            "I really like both.":
                $ persistent.jn_player_tea_coffee_preference = "both"
                n 1uwdaw "Really?{w=0.1} Huh."
                n 1nllaj "Most people I've met have a stronger preference for one or the other."
                n 1uslaj "Although...{w=0.3}"
                extend 1usqct " That sounds like you might be kind of a caffeine addict, [player]."
                n 1kchss "Or maybe not--{w=0.2}some kinds of tea are more suited for helping you sleep, now that I think about it."
                n 1uchgn"Well, either way,{w=0.1} good to know you're not picky!"
                
```
One more branch, just in case. :P I'm not sure if we also wanna add a way for people to change their answer later?

Reply:
Sure, I can add another choice for both. I'll also take a look at the ability to change preference.